hydra-cluster/exe/hydra-cluster/Main.hs

```diff
@@ -30,18 +31,21 @@ run options =
             waitForFullySynchronized fromCardanoNode node
             publishOrReuseHydraScripts tracer node
               >>= singlePartyHeadFullLifeCycle tracer workDir node
+              >>= uncurry (checkObservationsCallback node)
         Nothing -> do
           withCardanoNodeDevnet fromCardanoNode workDir $ \node -> do
             txId <- publishOrReuseHydraScripts tracer node
-            singlePartyOpenAHead tracer workDir node txId $ \client walletSk -> do
-              case scenario of
-                Idle -> forever $ pure ()
-                RespendUTxO -> do
-                  -- Start respending the same UTxO with a 100ms delay.
-                  -- XXX: Should make this configurable
-                  respendUTxO client walletSk 0.1
+            let
+            singlePartyOpenAHead tracer workDir node txId (checkObservationsCallback node) $
```

Comment:
Why are we doing this? This seems completely unrelated to the goal "Add blockfrost mode to hydra-chain-observer".